support/windows-server/system-management-components/PowerShell Console characters garbled for CJK languages

```diff
@@ -0,0 +1,41 @@
+---
+title: PowerShell Console characters garbled for Chinese, Japanese and Korean languages on Windows Server 2022
+description: PowerShell Console characters garbled for Chinese, Japanese and Korean languages on Windows Server 2022.
+ms.date: 08/13/2021
+author: kryalava
+ms.author: kryalava 
+manager: dcscontentpm
+audience: itpro
+ms.topic: troubleshooting
+ms.prod: windows-server
+localization_priority: medium
+ms.reviewer: slee
+ms.prod-support-area-path: PowerShell
+ms.technology: windows-server-system-management-components
+---
+# PowerShell console characters are garbled for Chinese, Japanese and Korean languages on Windows Server 2022
+
+This article provides a resolution to an issue that characters in PowerShell console getting garbled.
+
+_Applies to:_ &nbsp; Windows Server 2022
+_Original KB number:_ &nbsp;
+
+## Symptom
+
+When you launch PowerShell console the characters are getting garbbled. This applies only to Chinese, Japanese and Korean languages SKUs.
+
+## Cause
+
+Windows PowerShell .lnk shortcut is hardcoded to Consolas font, by default. This font doesn't have the glyphs for CJK characters so they aren't rendered correctly. Changing to MSGothic explicitly will fix this as MSGothic has glyphs for CJK. The reason cmd.exe works is that the cmd .lnk shortcut doesn't specify a font, so the console chooses the right one at runtime depending on your language. A similar fix is being made in Windows, but won't be backported.
```

Comment:
Should specify which version of Windows will get the fix?

Reply:
I will add these details.